lib/adapter.js

```diff
@@ -37,74 +41,83 @@ class CucumberAdapter {
         this.capabilities = capabilities
 
         this.cucumberOpts = Object.assign(DEFAULT_OPTS, config.cucumberOpts)
```

Comment:
@lgandecki 

Ya, understand it was not merged yet. I may want to use webdriverIO with cucumber 2 for a company project (pending boss approval :D). 

Was using protractor with cucumber. You can have a look at https://github.com/protractor-cucumber-framework/protractor-cucumber-framework/blob/master/index.js if you want. I didn't read the code in too much detail yet, but it seem to work well using cucumber Cli for protractor with other hooks. Hope it helps :)